src/origin_sdk/gql/queries/input_queries/config.py

```diff
@@ -54,18 +55,43 @@
 def get_endo_exo_param_list_from_config(tech_config: Any):
     pd = tech_config.get("definitionParameters")
     ppy = tech_config.get("parameters")
-    exo_params = [
-        param.get("name") for param in ppy if param.get("appliesToExo") is True
+    exo_params_yearly = [
+        param.get("name")
+        for param in ppy
+        if param.get("appliesToExo") is True
+        and param.get("isMonthlyRegionTechParameter", False) is False
```

Comment:
![image](https://github.com/AuroraEnergyResearch/aurora-origin-python-sdk/assets/5574633/3ef9cf74-c5dc-4cb8-8aa4-2a34e1fb3a79)

This is subjective, but I don't like fallbacks for `get` personally, if you don't need one.

Reply:
TBH, I don't need default values here. Cause I have made it a default field in model-inputs. So I'll remove the default args.